elementary/monitor/dbt_project/packages.yml

```diff
@@ -1,5 +1,5 @@
 packages:
   - package: dbt-labs/dbt_utils
     version: [">=0.8.0", "<0.9.0"]
-  - git: https://github.com/elementary-data/dbt-data-reliability.git
```

Comment:
Can you leave this in comment? And also leave a comment to leave this commented 😅 
I keep going to dbt's docs to re-add this.